src/Abstractions/NexusMods.Abstractions.Loadouts/IApplyService.cs

```diff
@@ -15,12 +17,12 @@ public interface IApplyService
     /// </summary>
     /// <param name="loadoutId"></param>
     /// <returns></returns>
-    public Task<Loadout> Apply(LoadoutId loadoutId);
+    public Task<Loadout.Model> Apply(Loadout.Model loadout);
```

Comment:
The doc-string of this method needs to be updated. Also, does this even need a return value? Looking at the implementation, we just return the input...

Reply:
The idea is that we return the updated loadout, if we made any chages, but nothing in the code looks at it. Removed